storage/blockchain/src/config/mod.rs

```diff
@@ -40,5 +40,6 @@
 //! # Ok(()) }
 //! ```
 
+#[allow(clippy::module_inception)]
 mod config;
 pub use config::{Config, ConfigBuilder, ReaderThreads};
```

Comment:
This one seems like a mistake, I think I did this when I moved `ReaderThreads`, we could move the `config` sub-module into this module.